aws-java-eks-minimal/app/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id 'application'
+}
+
+repositories {
+    mavenLocal()
+    maven { // The google mirror is less flaky than mavenCentral()
+        url("https://maven-central.storage-download.googleapis.com/maven2/")
+    }
+    mavenCentral()
+}
+
+dependencies {
+    implementation "com.pulumi:pulumi:(,1.0]"
```

Comment:
this might be problematic, we'll se

Reply:
We do need a floating reference I think since we want this to use "latest" but perhaps there's a better way to do it?